docs/source/blog/neuroblueprint.md

```diff
@@ -0,0 +1,182 @@
+---
+blogpost: true
+date: 25 April 2024
+author: Joe Ziminski, Niko Sirmpilatze
+location: London, UK
+category: Blog
+language: English
+image: 1
+---
+
+# Data organisation with NeuroBlueprint
+*The challenge of unstandardised data in systems neuroscience*
+
+**estimated reading time: 10 minutes**
+
+```{image} /_static/blog_images/neuroblueprint/errant-science-data-organisation.jpg
+:align: center
+:width: 75%
+```
+
+The increasing use of software to automate acquisition and 
```

Comment:
Can a blank line be added above this text? It kinda looks like it's the caption.